include/hse/hse.h

```diff
@@ -308,6 +328,21 @@ hse_kvdb_mclass_info_get(struct hse_kvdb *kvdb, enum hse_mclass mclass, struct h
 bool
 hse_kvdb_mclass_is_configured(struct hse_kvdb *kvdb, enum hse_mclass mclass);
 
+/** @brief Reconfigure a media class storage path for an existing offline KVDB.
+ *
+ * @note This function is not thread safe.
+ *
+ * @param kvdb_home: KVDB home directory.
+ * @param mclass: media class to reconfigure
```

Comment:
Start the parameter definitions with a capital letter and end in a period.

Reply:
IMO arg decl comments should not exceed 100 columns in which case a period is unnecessary (put longer comments after all the decsl).